.github/workflows/prettier-html.yml

```diff
@@ -0,0 +1,37 @@
+name: Pretty HTML
+
+on:
+  workflow_dispatch:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gh-pages branch
+        uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+
+      - name: Find and Remove </source> Tags
+        run:  find . -type f -name "*.html" -exec sed -i 's/<\/source>//g' {} +
+      
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

Comment:
Let's use the [latest version](https://github.com/actions/setup-node?tab=readme-ov-file#usage) `v4` here.